fineract-provider/src/main/java/org/apache/fineract/infrastructure/configuration/domain/ConfigurationDomainServiceJpa.java

```diff
@@ -444,4 +444,11 @@ public boolean isBusinessDateEnabled() {
     public boolean isCOBDateAdjustmentEnabled() {
         return getGlobalConfigurationPropertyData(ENABLE_AUTOMATIC_COB_DATE_ADJUSTMENT).isEnabled();
     }
+
+    @Override
+    public boolean isReversalTransactionAllowed() {
+        final String propertyName = "allow-reversals-for-reversed-transactions";
```

Comment:
This configuration is misleadingly named. As far as I can see, you touched FixedDepositAccounts, RecurringDepositAccount and SavingsAccounts.
Please name the config accordingly cause judging by its name it should also configure all other type of accounts, for example Loan accounts.

Reply:
I changed configuration name to `enable-post-reversal-txns-for-reverse-transactions`